.eslintrc

```diff
@@ -31,6 +31,7 @@
     "quotes": ["error", "single", { "avoidEscape": true }],
     "ramda/always-simplification": ["error"],
     "ramda/compose-simplification": ["error"],
+    "ramda/complement-simplification": "off",
```

Comment:
Can we turn this off just for the case of `nand` and  not globally ? 

```js
const nand = complement(and); // eslint-disable-line ramda/complement-simplification

Reply:
That makes way more sense, thank you!